src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -87,6 +88,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $this->convertController($route);
 
+            // Converts a short notation service:method to a class::method.
+            if ($route->hasDefault('_controller') && 'txt' === $input->getOption('format')) {
+                list($service, $method) = explode(':', $route->getDefault('_controller'), 2);
```

Comment:
Be careful with this, it's not always `service:method` pattern, when a short notation controller (`a:b:c`) is provided, stuff as this could happen:

```yml
# services.yml
services:
    appbundle:
        class: stdClass
```

```php
list($service, $method) = explode(':', 'AppBundle:Default:index', 2);
// $service <<< 'AppBundle'
// $method  <<< 'Default:index'

// so 
$this->getContainer()->get($service) //case insensitive
// return a valid service "appbundle"
```
and then shows a wrong output:
```bash
bin/console debug:router index
+--------------+---------------------------------------------------------+
| Property     | Value                                                   |
+--------------+---------------------------------------------------------+
| Defaults     | _controller: stdClass:Default:index                     |
|              |    (service: AppBundle)                                 |

```

you may need to check if it's the `service:method` pattern before.